common/types/activation_test.go

```diff
@@ -42,21 +42,15 @@ func unsafeMul(x, y uint64) uint64 {
 	return x * y
 }
 
-func layerTester(tb testing.TB, encodable codec.Encodable) LayerID {
```

Comment:
this is not sustainable. 
i know you don't like helper functions. but other teammates do.
and we get in the cycle of removing/creating helper functions endlessly.
i think we should keep this.

Reply:
The only place where this function would be used now is for the test right below it.

The other 3 places are in different packages now. I would now need to create a new package that contains this function, just so it can be re-used.

While I'm actually of the opinion to get rid of those tests. All 4 tests do the same thing: check if go-scale encodes and decodes correctly, once with an ATX, once with a Proposal, once with a Ballot and once with a Block. Why should this fail with one of those types and not with the others?

Reply:
because those types can be changed separately later. isn't that the idea of unit-testing?

Reply:
I think I was able to create something that can be re-used using generics. See my latest commit.